src/nashpy/integer_pivoting/integer_pivoting.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 
 
-def make_tableau(M):
+def make_tableau(M: np.ndarray) -> np.ndarray:
```

Comment:
Let us change this to:

```suggestion
def make_tableau(M: npt.ArrayLike) -> npt.ArrayLike:
```

This will require you to add:

```
import numpy.typing as not
```

in the imports.

Reply:
This is more in line with the current guidance about numpy typing I believe. https://numpy.org/doc/stable/reference/typing.html#numpy.typing.NDArray